### PR TITLE
feat(clients/go): implement activate jobs command

### LINKED ISSUE #1484: Go client can activate jobs
- Activate jobs by type with amount

### PR DESCRIPTION
**Pull request by menski**
closes #1484

### CODE REVIEW

File: clients/go/commands/activateJobs_command.go [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+type ActivateJobsCommandStep1 interface {
+	JobType(string) ActivateJobsCommandStep2
+}
+
+type ActivateJobsCommandStep2 interface {
+	Amount(int32) ActivateJobsCommandStep3
+}
+
+type ActivateJobsCommandStep3 interface {
+	DispatchActivateJobsCommand
```
Comment (Zelldon) [inline clients/go/commands/activateJobs_command.go]: What does this?

**Review by ChrisKujawa — APPROVED** [review]

File: clients/go/commands/activateJobs_command.go (hunk 1 above)
Comment (menski) [reply clients/go/commands/activateJobs_command.go]: https://golang.org/doc/effective_go.html#embedding

Comment (Zelldon) [reply clients/go/commands/activateJobs_command.go]: Thanks :+1:

**Pull request merged**